canonical/snapcraft.io
# Allow screenshot reordering on listing page

## Done
Enabled reordering of screenshots on the snap listing page

## QA
- Go to https://snapcraft-io-4083.demos.haus/SNAP_NAME/listing?show_react_listing=true
- Check that you can reorder screenshots, then save and they are saved in the correct order

## Issue
Fixes https://github.com/canonical/snapcraft.io/issues/4081
